app/src/components/common/TmSessionImport.vue

```diff
@@ -61,14 +61,16 @@ import FieldSeed from "common/TmFieldSeed"
 import SessionFrame from "common/SessionFrame"
 import { mapGetters } from "vuex"
 import Steps from "../../ActionModal/components/Steps"
-import { isHex } from "@polkadot/util"
 import { mnemonicValidate } from "@polkadot/util-crypto"
 
 const has12or24words = (param) => {
   return (
     param && (param.split(` `).length === 12 || param.split(` `).length === 24)
   )
 }
+const isHex = value => {
+  return /0x[0-9a-f]+/i.test(value)
+}
```

Comment:
whhhhaaaa...? Are we leveling up and importing with our minds or what is going here

Reply:
![](https://media1.giphy.com/media/1jnyRP4DorCh2/giphy.gif)